care/emr/resources/service_request/spec.py

```diff
@@ -203,7 +203,7 @@ class ServiceRequestReadSpec(BaseServiceRequestSpec):
 
     @classmethod
     def perform_extra_serialization(cls, mapping, obj):
-        mapping["id"] = obj.external_id
+        mapping["id"] = obj.id
```

Comment:
> why?

It is a mistake in interpretation. Before mapping `mapping["based_on"] = ServiceRequestReadSpec.serialize(obj.based_on).model_dump(` it was returning `id`  so after adding manual mapping it is returning `exterbal_id` I thought we needed to show `id`  instead of the `external_id`. Now I think I realised we are supposed to external_id  🙂